package-lock.json

```diff
@@ -124,9 +124,9 @@
       }
     },
     "node_modules/@babel/cli": {
-      "version": "7.16.7",
-      "resolved": "https://registry.npmjs.org/@babel/cli/-/cli-7.16.7.tgz",
-      "integrity": "sha512-0iBF+G2Qml0y3mY5dirolyToLSR88a/KB6F2Gm8J/lOnyL8wbEOHak0DHF8gjc9XZGgTDGv/jYXNiapvsYyHTA==",
+      "version": "7.16.8",
```

Comment:
We didn't change anything in the package.json file. Not sure if it's needed

Reply:
package-lock.json is showing a fair number of changes. I looked at the current version of the development branch and it seems that these are real changes from the current code. Thus, if you didn't intend to change the packages then this change should be removed so the code can be tested with those versions of the packages. This also has to happen to allow merging of the PR since it can only have desired changes. If you need help then let me know.